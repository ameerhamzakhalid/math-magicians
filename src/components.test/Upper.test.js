import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Upper from '../components/Upper';

describe('Upper', () => {
  it('renders the correct text', () => {
    const { getByText } = render(<Upper total={1} operation="+" next="2" />);
    const input = getByText('1 + 2');
    expect(input).toMatchSnapshot();
  });
});
