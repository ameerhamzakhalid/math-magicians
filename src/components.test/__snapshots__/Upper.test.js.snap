// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Upper renders the correct text 1`] = `
<span
  class="input"
>
  1 + 2
</span>
`;
